pages/mydoc/notes.md

```diff
@@ -39,6 +39,35 @@ We will keep the discussions ongoing via web meetings to work out the details of
 
 **ENTRIES ARE IN REVERSE CHRONOLOGICAL ORDER**
 
+## Notes from the 6th Scattering WebEx Meeting (17 Oct 2018)
+
+PARTICIPANTS: Ryan - Eugene - Stefan - Davide - Ian - Annakaisa - Patrick - Robin
+
+[MEETING SLIDES](https://)
+
+1. Apparently there has been problems with the FTP server. It might be connected with some work on the network the IT department did in the past weeks. Currently the servers looks up, running and acceccible from either inside and outside the network. If the problem persists, please report to Davide or Stefan.
```

Comment:
I just learned that my office has been blocking FTP connections, so we can remove this.